test/univariate_bounds.jl

```diff
@@ -0,0 +1,88 @@
+using Distributions
+using Test
+
+# to make sure that subtypes provides the required behavoir without having to add
+# a dependency to InteractiveUtils
+function _subtypes(m::Module, x::Type, sts=Base.IdSet{Any}(), visited=Base.IdSet{Module}())
```

Comment:
the function here is non-trivial and does some heavy reflection, could we document it a bit more (takes input this and that, returns that)

Reply:
This function is being used during testing only and I don't know how much time I want to spend documenting functions that won't be used outside of a single test file.  [Here is the implementation from InteractiveUtils.jl](https://github.com/JuliaLang/julia/blob/c6da87ff4bc7a855e217856757ad3413cf6d1f79/stdlib/InteractiveUtils/src/InteractiveUtils.jl#L175).  In addition, any documentation of this function should really be an upstream issue and doing that work here seems redundant.

Reply:
Ok with me.

Reply:
yeah it was more for maintainers to know what to do with this and what to expect from it, even some few months from now